src/App.vue

```diff
@@ -26,11 +31,25 @@ export default {
 
   methods: {
     addTodo (newTodo) {
-      this.todos.push(newTodo)
+      let todo = {
+        title: newTodo,
+        completed: false,
+        editing: false
+      };
+      if (newTodo != '') {
```

Comment:
please do not send `add event` if the title is empty

Reply:
Fixed. Check moved to AddNewTod-component.